### DISCUSSION TITLE
Occasional variability

### DISCUSSION THREAD

**Discussion by vskantze** [nlmixrdevelopment/nlmixr#653] [Q&A]
Hi,

I have data with multiple responses per individual spread over different days and I would like to have predictions for all of these responses and not just one prediction for all. 
Is there a way to include occasional variability in the model, i.e., random effects on an occasional variable to achieve this? 

Thx!

**Comment by mattfidler** [comment]
See https://github.com/nlmixrdevelopment/nlmixr/issues/93#issuecomment-2093486042
